requirements/pmdarima-requirements.txt

```diff
@@ -1 +1 @@
-pmdarima>=1.8.0
```

Comment:
Why can't we support 1.8.0 anymore?

Reply:
1.8.2 and 1.8.3 have some pretty nasty bugs in them (acf function throws with defaults  and the provided demo datasets in pmdarima, for instance). 1.8.5 was the only minor version that didn't have some sort of bug that was causing test failures. The original version of Diviner just happened to be using 1.8.5 by virtue of timing.

Reply:
Sounds good! :D